src/call.c

```diff
@@ -2212,17 +2233,16 @@ int call_accept(struct call *call, struct sipsess_sock *sess_sock,
 		mem_deref(rid);
 	}
 
-	err = sipsess_accept(&call->sess, sess_sock, msg, 180, "Ringing",
+	err = sipsess_accept(&call->sess, sess_sock, msg, 100, "Trying",
 			     account_rel100_mode(call->acc),
 			     ua_cuser(call->ua), "application/sdp", NULL,
 			     auth_handler, call->acc, true,
 			     sipsess_offer_handler, sipsess_answer_handler,
 			     sipsess_estab_handler, sipsess_info_handler,
 			     call->acc->refer ? sipsess_refer_handler : NULL,
 			     sipsess_close_handler,
-			     call, "Allow: %H\r\n%H",
-			     ua_print_allowed, call->ua,
-			     ua_print_require, call->ua);
+			     call, NULL);
```

Comment:
This changes the default behavior of the baresip core. Whereas, the bevent solution will keep the default behavior and allow the application to reject an incoming call.

Reply:
Is it okay to allocate a SIP session without sending any reply? If so, is it possible that we use `sipsess_alloc()` here directly?

An API function called `call_accept()` should send a reply in my opinion.